### PR TITLE
Add support for coverage_qc_report - fix #987.

### LINKED ISSUE #987: cant see coverage report
hi,

I have created an issue in the "orderportal" of github aswell, but I thought I would create on here as well..... 

We do not see the coverage report from the samples uploaded in scout. I have been told to create an issue on here regarding this issue. Please see below.

image

### PR DESCRIPTION
**Pull request by dnil**
## Description
This PR adds support for capturing and exporting BALSAMIC coverage reports from analysis to Scout

### How to prepare for test
- [x] ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
See https://github.com/Clinical-Genomics/hermes/pull/13

### Expected test outcome
- [x] check that ...
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by
- [x] tests executed by
- [x] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [x] Deploy this branch
- [ ] Inform to production and demo changes

### CODE REVIEW

**Comment by hassanfa** [comment]
fyi, `balsamic 6.1.2` deployed and support this.

**Comment by dnil** [comment]
Thank you Hassan! We (that is @Mropat) will make a judgment call on wether to rerun or if we should just fake the tag in for testing.

**Comment by hassanfa** [comment]
> Thank you Hassan! We (that is @Mropat) will make a judgment call on wether to rerun or if we should just fake the tag in for testing.

Sure, if you need a different tag, just let me know. balsamic change is tiny.

File: cg/meta/upload/scout/hk_tags.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ class CaseTags(BaseModel):
     peddy_check: Set[str] = Field(None, description="Peddy pedigree check, only for rare disease")
     multiqc_report: Set[str]
     delivery_report: Set[str]
+    coverage_qc_report: Set[str] = Field(None, description="Static coverage report for cancer")
```
Comment (moonso) [inline cg/meta/upload/scout/hk_tags.py:24]: This is same/similar to delivery_report for rare disease so I suggest that we reuse that field here

**Comment by hassanfa** [comment]
btw, when the tag is decided, please open a PR similar to https://github.com/Clinical-Genomics/BALSAMIC/pull/568 in BALSAMIC for the tag you'd like to have.

**Comment by Mropat** [comment]
image

**Review by Mropat — APPROVED** [review]

**Comment by Mropat** [comment]
Installed on prod:
```
Building wheels for collected packages: cg
  Building wheel for cg (PEP 517) ... done
  Created wheel for cg: filename=cg-20.1.4-py3-none-any.whl size=398792 sha256=d99648b0b02ff30557454f3f31ee54b5b60c9018de47cb1323b6a9de5c1e004d
  Stored in directory: /home/maryia.ropart/tmp/pip-ephem-wheel-cache-rg8dtciv/wheels/8c/1e/b8/816ed4facf8c5628b55966d3f2448eeb6ca70f35deff59389e
Successfully built cg
Installing collected packages: cg
  Attempting uninstall: cg
    Found existing installation: cg 20.1.3
    Uninstalling cg-20.1.3:
      Successfully uninstalled cg-20.1.3
Successfully installed cg-20.1.4
```

@ashwini06 @keyvanelhami @karlnyr @emmser @henningonsbring
Now the report is included in scout config when delivering to Scout. The report will show up under Reports / Delivery in scout case view

**Pull request merged**